official/resnet/resnet_run_loop.py

```diff
@@ -384,16 +405,37 @@ def input_fn_eval():
       benchmark_logger = logger.BenchmarkLogger(flags.benchmark_log_dir)
       benchmark_logger.log_estimator_evaluation_result(eval_results)
 
+  # Return the classifier after all epochs are completed.
+  return classifier
+
+
+def export_savedmodel(classifier, export_dir, shape, batch_size):
```

Comment:
savedmodel seems to be a two words to me, so probably export_saved_model

Reply:
Unfortunately, there is a lot of inconsistency in the spelling of this across TF+friends. In this case, I went with what Estimator has, which is export_savedmodel.